luaui/configs/gridmenu_layouts.lua

```diff
@@ -250,6 +250,10 @@ local unitGrids = {
 	legassistdrone = {
 		{
 			{ "legmex", "legsolar", "legwin", },                -- mex, solar, wind
+			{ "cormakr", "", "coruwms", "legtide"},             -- T1.5 mex, uw m storage, tidal
```

Comment:
I think this is broken now, it contains duplicates and should keep to 4 rows max if I'm not mistaken?